platform/kubernetes/wrapper/wrap.go

```diff
@@ -0,0 +1,85 @@
+package wrapper
```

Comment:
What is the purpose of this program?

Reply:
This library is used by various launcher programs to generate the kubeconfig needed to connect to the cluster, or to grab other information relating to launching and wrapping kubernetes or other components.